packages/marble-api/openapis/marblecore-api.yaml

```diff
@@ -3269,6 +3271,126 @@ paths:
           $ref: '#/components/responses/403'
         '404':
           $ref: '#/components/responses/404'
+  /scenario-testrun:
+    get:
+      tags:
+        - TestRun
+      summary: List all test runs for a scenario
+      operationId: listTestRuns
+      security:
+        - bearerAuth: []
+      parameters:
+        - name: scenarioId
+          description: ID of the scenario that need to be fetched
+          in: query
+          required: true
+          schema:
+            type: string
+            format: uuid
```

Comment:
Since we do need to GET testruns by ID, I changed the signature from
```
GET /scenario-testrun/{scenarioId}
```
To
```
GET /scenario-testrun?scenarioId=... // use of a query param
```

Let me know if it's ok.

Another solution is to move everything link to a testrun under something like
```
GET /scenario/{scenarioId}/testrun
POST /scenario/{scenarioId}/testrun
...
```
But it doesn't make sens to fill the `{scenarioId}` on API endpoint using `{testRunId}`

Reply:
the changed signature is ok for me. for the moment let's keep it simple, we will be able to change of needed. I will adapt the backend accordingly